LICENSE.md

```diff
@@ -0,0 +1,22 @@
+Copyright (c) 2018-2020 Michalis Zampetakis
+Copyright (c) 2015-2020 Thanasis Zografos
+Copyright (c) 2015-2020 Antonios Chariton
+Copyright (c) 2015-2020 Nikos Kampitakis
+Copyright (c) 2019-2020 Giannis Koutsaftakis
+Copyright (c) 2016-2020 Yorgos Saslis
+Copyright (c) 2015-2020 Paschalis Padeleris
+Copyright (c) 2015-2020 Foivos Zakkak
+Copyright (c) 2018-2020 Giannis Vrentzos
+Copyright (c) 2015-2020 Fotis Adamakis
```

Comment:
Hmmm this is probably some kind of standard practice I don't know about, but ... why "2020" and not "2021"? 🤔

Reply:
Because that's when the PR was opened :D

Reply:
We could also have a generic "Copyright (c) 2015-2020 The DevStaff community" or similar.

Reply:
Done.